test_vgpu.sh

```diff
@@ -155,14 +193,18 @@ done
 END_OF_REQ_VGPU_TYPE
 )
 
-echo "###################Devstack start stack#####################"
 pushd $DEVSTACK_PATH/
 if ! grep "enabled_vgpu_types=" ${DEVSTACK_PATH}/local.conf; then
     echo "[devices]" >> ${DEVSTACK_PATH}/local.conf
     echo "enabled_vgpu_types = $first_vgpu_type" >> ${DEVSTACK_PATH}/local.conf
 fi
+echo "###################Set image handler to remote stream#####################"
```

Comment:
If we're doing this here, why do we also need the line at 109?

Reply:
Sorry, forgot to remove it. It doesn't work when I modify local.conf.